src/main/java/com/thealgorithms/ciphers/MonoAlphabetic.java

```diff
@@ -0,0 +1,44 @@
+package com.thealgorithms.ciphers;
+
+public final class MonoAlphabetic {
+
+    // Private constructor to prevent instantiation of utility class
+    private MonoAlphabetic() {
+        throw new UnsupportedOperationException("Utility class");
+    }
+
+    // Encryption method
+    public static String encrypt(String data, String key) {
+        StringBuilder sb = new StringBuilder();
+        data = data.toUpperCase();
```

Comment:
I would suggest to throw an exception if the input has lower case letters.

Reply:
Added the exception accordingly.